config/rbac/csr_creator_clusterrole.yaml

```diff
@@ -0,0 +1,14 @@
+# ClusterRole 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: csr-creator-clusterrole
```

Comment:
`ClusterRole` is a non-namespaced resource. We could update the name to something like`byoh-csr-creator-clusterrole` to explicitly say this belongs to `byoh`(or if you have other thoughts). Same applies to `ClusterRoleBinding`.